<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addCustom</class>
 <widget class="QDialog" name="addCustom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="raceRadial">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Race</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="relRadial">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Religion</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="itemRadial">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select what you wish to edit:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="classRadial">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Class</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="weaponRadial">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Weapon</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="featRadial">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Feat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Existing</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="armorRadial">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Armor</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="spellRadial">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Spell</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
